README.md

```diff
@@ -14,7 +14,7 @@ This sample showcases:
 
 ## Screenshots
 
-<img src="screenshots/sunflower.gif"/>
```

Comment:
Should we remove the `sunflower.gif`?

Reply:
I decided to leave it in because they're still currently indexed and served to pages, I'll do a follow up CL to remove them once the new screenshots have been indexed. A cron job runs daily for this, so conservative TTL is 36 hrs.